### PR TITLE
[RFR] A few small changes in changelog.md

### PR DESCRIPTION
**Pull request by SimonBiggs**
This is just a few small edits from my phone, that's fraut with potential for error though, so feel free not to merge if I've done something untoward.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
 All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-<br/>
+
```
Comment (Centrus007) [inline changelog.md]: I wanted to include a bit of space to clearly visually separate releases. Multiple line breaks weren't doing the job.

We can try merging this in so you can see on [pymedphys.com](http://pymedphys.com). If it doesn't look nice, we can revert :)

File: changelog.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 
 ## [Unreleased]
 
 ### Added
 - ProfileDose object built ontop of xarray. [@SimonBiggs](https://github.com/SimonBiggs).
 
-### Changed
-- **Breaking Change**: All uses of "dcm" in directory names, module names, function names, etc.
+### Breaking Changes
```
Comment (Centrus007) [inline changelog.md]: (Referring to #66) I agree, this is probably better.

Perhaps we should consider a whole new set of change headings...

**Comment by SimonBiggs** [comment]
Yup I think the following headings are helpful:

https://github.com/angular/angular/blob/master/CHANGELOG.md

**Comment by Centrus007** [comment]
Strictly speaking, not all of the changes that are now under **Breaking Changes** were "breaking" changes... haha

**Comment by Centrus007** [comment]
`angular` aren't particularly consistent with their headings, but it's a start :)

**Comment by SimonBiggs** [comment]
> Strictly speaking, not all of the changes that are now under Breaking Changes were "breaking" changes... haha

True. Oops, hmm, can I leave that to you to work out how best to categorise? Groceries are finished toilet stops complete :). About to be back on the road from Port Augusta :).

**Comment by Centrus007** [comment]
@SimonBiggs, the lack of `<br/>`s are not working. See the follwing images:

#### Without `<br/>`
ss1
<br/>
ss2

<br/>

#### With `<br/>`
ss3
<br/>
ss4

**Comment by Centrus007** [comment]
> True. Oops, hmm, can I leave that to you to work out how best to categorise?

Sure :)

**Pull request merged**